FBSnapshotTestCase/FBSnapshotTestController.m

```diff
@@ -126,25 +129,24 @@ - (BOOL)compareReferenceImage:(UIImage *)referenceImage
                     tolerance:(CGFloat)tolerance
                         error:(NSError **)errorPtr
 {
-  if (CGSizeEqualToSize(referenceImage.size, image.size)) {
-    BOOL imagesEqual = [referenceImage fb_compareWithImage:image tolerance:tolerance];
-    if (NULL != errorPtr) {
-      *errorPtr = [NSError errorWithDomain:FBSnapshotTestControllerErrorDomain
-                                      code:FBSnapshotTestControllerErrorCodeImagesDifferent
-                                  userInfo:@{
-                                             NSLocalizedDescriptionKey: @"Images different",
-                                             }];
-    }
-    return imagesEqual;
+  BOOL sameImageDimensions = CGSizeEqualToSize(referenceImage.size, image.size);
+  if (sameImageDimensions && [referenceImage fb_compareWithImage:image tolerance:tolerance]) {
+    return YES;
   }
```

Comment:
There is an [issue](https://github.com/facebook/ios-snapshot-test-case/issues/109) that I've been meaning to tackle for a while, do you mind incorporating in with PR? If the there is a tolerance set, for example 0.1 (10% IRRC) and the test fails because the difference is 17%, display this number in the error message.